README.md

```diff
@@ -924,6 +926,18 @@ kafka = Kafka.new(
 
 **NOTE**: It is __highly__ recommended that you use SSL for encryption when using SASL_PLAIN
 
+##### SCRAM
+Since 0.11 kafka supports [SCRAM](https://kafka.apache.org/documentation.html#security_sasl_scram). 
+
+```ruby
+kafka = Kafka.new(
+  sasl_scram_username: 'username',
+  sasl_scram_password: 'password,
+  sasl_scram_mechanism: Kafka::SCRAM_SHA256,
```

Comment:
I'd prefer a String name of the mechanism, e.g. `"sha256"`, as that will make it a lot easier to integrate with config files and the like.

Reply:
Ok. I can change that. Do you prefer `"sha256"` over `"SHA-256"` (the current value of the constant)?

Reply:
I prefer `"sha256"`.